libminifi/include/core/state/nodes/DeviceInformation.h

```diff
@@ -490,6 +477,8 @@ class DeviceInfoNode : public DeviceInformation {
   std::string hostname_;
   std::string ip_;
   std::string device_id_;
+  static utils::SystemCPUUsageTracker cpu_load_tracker_;
+  static std::mutex cpu_load_tracker_mutex_;
```

Comment:
this is static because CPU Usage tracking needs some persistent data, because it compares CPU ticks between heartbeats, but the response nodes are instantiated and destroyed with each heartbeat.